docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -117,3 +117,11 @@ Then install onnx through:
 pip install onnx
 ```
 Then reinstall mmdeploy.
+
+
+### FAQs
+
+- For devices with arch==7.2 , `#assertion/root/workspace/mmdeploy/csrc/backend_ops/tensorrt/batched_nms/trt_batched_nms.cpp,98 2022-02-08 07:15:09,334 - mmdeploy - ERROR - visualize tensorrt model failed.`
```

Comment:
the user might not understand what is arch==7.2, use jetson tx2, and jetson nano instead.
And we can add an assert in the nms plugin if arch==7.2.